packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js

```diff
@@ -23,7 +27,9 @@ export default class {
     this.DatabaseService = DatabaseService;
     this.Poller = Poller;
     this.NAME_PATTERN = NAME_PATTERN;
+    this.MIN_NAME_LENGTH = MIN_NAME_LENGTH;
     this.MAX_NAME_LENGTH = MAX_NAME_LENGTH;
+    this.regexp = new RegExp(this.NAME_PATTERN);
```

Comment:
Not already a regex ?

Reply:
You're right, I'll change this to use the pattern directly